tests/test_manager.py

```diff
@@ -10,6 +11,11 @@
 from xqueue_watcher import manager
 from tests.test_xqueue_client import MockXQueueServer
 
+try:
```

Comment:
I think that it would be better at this way:

```python
if six.PY2:	
    from StringIO import StringIO  # pylint: disable=import-error	
else:	
    from io import StringIO  # pylint: disable=import-error	
```

Reply:
Or just `from six import StringIO`